libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -57,10 +61,28 @@ type addDockerMetadata struct {
 	fields          []string
 	sourceProcessor processors.Processor
 
-	pidFields []string      // Field names that contain PIDs.
-	cgroups   *common.Cache // Cache of PID (int) to cgropus (map[string]string).
-	hostFS    string        // Directory where /proc is found
-	dedot     bool          // If set to true, replace dots in labels with `_`.
+	pidFields       []string      // Field names that contain PIDs.
+	cgroups         *common.Cache // Cache of PID (int) to cgropus (map[string]string).
+	hostFS          string        // Directory where /proc is found
+	dedot           bool          // If set to true, replace dots in labels with `_`.
+	dockerAvailable bool          // If Docker exists in env, then it is set to true
+}
+
+// isDockerAvailable checks if Docker is available in running environment
+func isDockerAvailable() bool {
+	cli, err := client.NewEnvClient()
+	errorMsg := fmt.Sprintf("%v: docker environment not detected.", processorName)
+	if err != nil {
+		logp.Info(errorMsg)
```

Comment:
Log this error at debug level, and with the error too?
```suggestion
		logp.Debugf("%s: %v", errorMsg, err)
```
And same thing for the other error some lines below.